external/js/kyber/spec/pairing/bn.spec.ts

```diff
@@ -0,0 +1,253 @@
+import { randomBytes } from 'crypto';
+import BN from 'bn.js';
+import { G1, G2, GT } from '../../src/pairing/bn';
+import { p } from '../../src/pairing/constants';
+
+/**
+ * Unsecure pseudo random generator
+ * @returns a big number between 0 and p
+ */
+function getRandomInt(): BN {
```

Comment:
I do not see why we should ever be shipping what we call an unsecure generator, even in test code. In test code, if we generate keys from some static seed, that's insecure, but there's a reason for it (test repeatability).

I propose that in the one test where there are used that either you use hardcoded scalars (i.e. 1, 2, 3) or you get real cryptographically random input.

It is important, especially because students use our code, that we do not teach bad habits.